phonebook.code-workspace

```diff
@@ -38,14 +38,13 @@
     }
   ],
   "settings": {
-    "cSpell.words": [
-      "Kubernetes",
-      "Phonebook"
-    ],
+    "cSpell.words": ["Kubernetes", "Phonebook"],
     "javascript.preferences.quoteStyle": "single",
     "typescript.preferences.quoteStyle": "single",
     "javascript.referencesCodeLens.enabled": true,
     "typescript.referencesCodeLens.enabled": true,
+    "prettier.disableLanguages": [],
+    "editor.formatOnSave": true,
```

Comment:
Sollte nicht mit drin sein @DanielHabenicht ?

Reply:
Das zweite würde ich zur Diskussion stellen, da aber irgendwie niemand formatiert fände ich es nicht schlecht. 
Das erste brauch man weil wir prettier für eigentlich alles verwenden und menschen (wie ich) persönlich den Formatierer für html dateien ausgestellt haben und prettier in diesem fall dann nicht mehr in VSCode funktioniert.